### ISSUE TITLE
[Feature Request] Find and Replace Texts in editor

### ISSUE THREAD

**Issue by JohnWang1023** [zadam/trilium#679]

**Comment by zadam** [comment]
Just to give some answer here - unfortunately this is pretty difficult to do in Electron so I don't see this happening soon even though this would be a useful feature of course.

**Comment by JohnWang1023** [comment]
Possible to integrate in the code editor using https://codemirror.net/demo/search.html ?

**Comment by zadam** [comment]
the annoying thing is that this would be specific to code editor and not consistent with text editor ...

**Comment by andersmmg** [comment]
Any updates on this seeing as it's supported in CKEditor 5?
https://ckeditor.com/docs/ckeditor5/latest/features/find-and-replace.html

**Comment by perfectra1n** [comment]
This would be awesome!

**Comment by zadam** [comment]
This is actually part of the toolbar:

![](https://github.com/zadam/trilium/assets/617641/686c56e2-bf5d-4e75-ad6f-9598e442f38a)

Unfortunately it is not possible to bind this to a keyboard shortcut :-(

**Comment by perfectra1n** [comment]
Oh I didn't even know that was there!

Maybe it's possible to also include something like it in the little pane that pops up with `Ctrl + F`?:
![image](https://github.com/zadam/trilium/assets/2390633/2f1c36be-edb9-40a0-8aca-d1a1e2936edd)

**Comment by zadam** [comment]
Yes, that would be nice, but unfortunately not trivial to implement since it would have to hook into CKEditor/CodeMirror APIs. Still I would like to get to it at some point.

**Comment by Meroyanae** [comment]
Hi, is there a way to perform a search/replace by editing code, especially HTML code?
